src/fileio.cpp

```diff
@@ -16,25 +16,34 @@ vector<vector<float>> getCorners (float x, float y, float z, float s) {
   };
 }
 
-vector<vector<vector<float>>> getTriangles (vector<vector<float>> corners) {
+vector<vector<vector<float>>> getTrianglesAndNormal (vector<vector<float>> corners) {
```

Comment:
Yeah, I'll change that. This name is a remnant from my previous attempt, where I tried storing a single number for each normal (0/1/2 for x/y/z).